BioSQL/.flake8

```diff
@@ -12,4 +12,10 @@ ignore =
     # D103	Missing docstring in public function
     # D104	Missing docstring in public package
     # D105	Missing docstring in magic method
-    D101,D102,D103,D105
+    D101,D102,D103,D105,
+    # ================================================
+    # flake8 whitespace: W50#
+    # ================================================
+    # W503	Line break before binary operator
+    # PEP8 is now in favour of line-breaks before binary operators
+    W503
```

Comment:
The W* codes are part of the flake8 core, so can you put this first on the list?